ci/citest/acceptance-test/multibox/10.0.100.17-vdc-executor-qemu-ovs/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export ENV_ROOTDIR="$(cd "$(dirname $(readlink -f "$0"))/.." && pwd -P)"
+export NODE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TMP_ROOT="${NODE_DIR}/tmp_root"
+
+. "${ENV_ROOTDIR}/config.source"
+. "${NODE_DIR}/vmspec.conf"
+. "${ENV_ROOTDIR}/ind-steps/common.source"
+
+IND_STEPS=(
+    "box"
+    "ssh"
+    "hosts"
+    "disable-firewalld"
+    "epel"
+    "openvswitch"
+    "qemu"
+    "kvm"
```

Comment:
In my opinion that would be a premature optimization. I don't think we're likely to implement Xen or other qemu features any time soon so I would merge the steps to keep the code simple.

We don't always have to do things my way though so if @toros11 agrees with you, I'm ok with leaving it this way.